### ISSUE TITLE
[Feature request] List playlist songs

### ISSUE THREAD

**Issue by wa7eedem** [maddox/itunes-api#30]
hi , is there a way to list all songs in a playlist ex : `GET : /playlists/:id`

thank you for a grate api

**Comment by eiGelbGeek** [comment]
Yes that is possible, but needs a major update of the code. I'll take a look at that when I'm done with the webradio integration in my fork.
